src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+div {
+  display: block;
```

Comment:
`div` block element.
this is a default value for `div`, no need to write it.
https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements